README.md

```diff
@@ -89,6 +89,55 @@ RUN pip install --no-cache-dir <A_REQUIRED_PACKAGE_ON_PYPI>
 USER 1001
 ```
 
+### Add Sanic extensions
```

Comment:
This is great docs, however how about moving this into the official [action server docs](https://github.com/RasaHQ/rasa/tree/main/docs/docs/action-server)? You'd need to open a PR in the rasa repo, that's where the sdk docs live now. Since this is a new feature I'd say, the docs PR should target `main` also.

Reply:
We probably need also a section on how this can be achieved in cases of different action servers implemented in a different programming language.

Reply:
Yeah, sounds great for the docs, also i was thinking of creating an example action server project using rasa_sdk and adding several extensions like background jobs, listeners that creates a database object and caches it.

Regarding the action servers in a different programming language, won't they be creating their own application server for themselves?

Reply:
> Regarding the action servers in a different programming language, won't they be creating their own application server for themselves?

You're right, probably also a different dependency will be required from that particular language ecosystem.

Reply:
@ancalita - i moved the docs to this branch: https://github.com/RasaHQ/rasa/blob/action-server-docs-plugins/docs/docs/action-server/add-sanic-extensions.mdx

Also is it possible to backport this release to `rasa-sdk=3.4.x` or will it go into rasa-sdk=3.5.x` ??
Let me know if this looks good. I can make the PR for docs targeting main.

Reply:
Great, feel free to open a PR, we might need to add this new section somewhere else in the docs config, can't remember off the top of my head.

>  is it possible to backport this release to rasa-sdk=3.4.x or will it go into rasa-sdk=3.5.x` 

This is technically adding new functionality to rasa-sdk, so it will have to be released in the next minor `3.6.0`.

Reply:
yes in the `sidebar.js`. Already did that 😄  

okay!! is there any issue if someone uses `rasa-sdk=3.6.0` but on an older rasa version like `3.4.x`??

Reply:
I don't think there's any issue apart from potentially poetry / pip not resolving dependencies according to the constraint in rasa 3.4.x 😅